apps/ledger-live-mobile/src/components/Skeleton.tsx

```diff
@@ -1,19 +1,26 @@
 import React, { memo, useRef, useEffect, useMemo, ReactElement } from "react";
 import { Animated } from "react-native";
 import { useTheme } from "@react-navigation/native";
+import styled from "styled-components/native";
+import { BaseStyledProps } from "@ledgerhq/native-ui/components/styled";
+import { BorderProps } from "styled-system";
 
-type Props = {
-  style?: any;
-  loading: boolean;
-  children?: ReactElement;
-  animated?: boolean;
-};
+type Props = BaseStyledProps &
+  BorderProps & {
+    style?: any;
+    loading: boolean;
+    children?: ReactElement;
```

Comment:
Seems like the more complete type for children is this : 
`children?: React.ReactNode;`